test/unit/helpers/JsonBuilder.ts

```diff
@@ -423,15 +418,16 @@ function getElementOfArray<T>(object: T | T[], index = 0): T {
   }
 }
 
-function updateBpmnElement<T extends TBaseElement | DiagramElement>(parentElement: T | T[], childElement: T, setValue: (value: T | T[]) => void): void {
-  if (parentElement) {
-    if (!Array.isArray(parentElement)) {
-      setValue([parentElement, childElement]);
+function getBpmnElementNewValue<T extends TBaseElement | DiagramElement>(currentValue: T | T[], elementToAdd: T): T | T[] {
```

Comment:
**question:** can we rename `currentValue` into `currentElement`?
As far as I understand we manipulate BPMN elements here.

The usage of "value" in the method name is unclear to me.
Perhaps, `enrichBpmnElement`? In this case, the first parameter could be simply named `element.